docs/Developers/Meta-review.rst

```diff
@@ -0,0 +1,140 @@
+They are used by people to publicly express their feelings about review
+comments, which provides feedback to comment authors and all other members.
+
+There are 6 types of reactions on GitHub:
+
+============  =============  =======  =====
+ Name         Markdown       Unicode  Emoji
+============  =============  =======  =====
+ THUMBS_UP    ``:+1:``       U+1F44D  👍
+ THUMBS_DOWN  ``:-1:``       U+1F44E  👎
+ LAUGH        ``:smile:``    U+1F604  😄
+ HOORAY       ``:tada:``     U+1F389  🎉
+ CONFUSED     ``:confused``  U+1F615  😕
+ HEART        ``:heart:``    U+FE0F   ❤️
+============  =============  =======  =====
+
+These reactions are reviews to reviews, thus we call them meta-reviews. To
+encourage people to do meta-reviews, we build a meta-review system, which
+collects reactions, analyze them, and use that information to score people.
+After scoring, a ranking list is shown on coala community website.
+
+At the moment, only THUMBS_UP and THUMBS_DOWN are collected and analyzed,
+as they are most commonly used and express strong and clear feelings:
+THUMBS_UP means the review comment is well-written and helpful, while
+THUMBS_DOWN means the review comment is misleading, or even worse, violates
+`coala Community Code of Conduct <https://github.com/coala/cEPs/blob/master/cEP-0006.md>`__.
+
+Meta-review Process
+-------------------
+
+The meta-review process for coala is as follows:
+
+1. Reviewers create their reviews.
+
+2. Anyone especially author of the pull request can do meta-review (THUMBS_UP,
+   THUMBS_DOWN) via reactions on GitHub.
+
+3. Meta-review information will be collected and analyzed automatically:
+
+   * Meta-reviewers will be given bonus points to encourage such behaviour
+
+   * Meta-reviewees will be given positive points (on receiving THUMBS_UP) or
+     negative points (on receiving THUMBS_DOWN). Those points are weighted
+     according to score of meta-reviewers. Those who have higher score in
+     the meta-review system will have more impacts on others, so please do
+     meta-reviews carefully.
+
+.. note::
+
+    Don't edit or even delete your review after it has been meta-reviewed.
+    If you do that, you are destroying feedback from the community. This
+    is considered as an improper behaviour and a negative score will be
+    given to you.
+
+Automated Scoring Process
+-------------------------
+
+A complete ranking list can be found on
+`Meta-review score ranking list <https://deploy-preview-143--coala-community.netlify.com/meta-review/>`__.
+Score is based on number of positive (THUMBS_UP) & negative (THUMBS_DOWN)
+reactions one receives, and number of positive & negative reactions one gives
+away. Calculation details can be found in
+`meta_review/handler.py <https://github.com/li-boxuan/community/blob/2143fbcf925beb1ffe5ab1d9dd4d177e827dece2/meta_review/handler.py>`__.
+
+Briefly, score is calculated based on formulas as follows:
```

Comment:
as it will probably change, it is better for this document to not include the score calculation details; links to the .py code and the cEP are much more useful and much more maintainable.

If your detailed explanation here is better than in the cEP, update the cEP ;-)

Reply:
I'll update the cEP after https://github.com/coala/community/pull/143 is merged ;)